.travis.yml

```diff
@@ -233,6 +232,7 @@ before_install:
 
         conda info -a;
         conda create -q -n test-environment python=$TRAVIS_PYTHON_VERSION pip ${TEST_OPT_DEPENDENCY};
+        pip install wurlitzer;  # package for capturing C level output (tests for stdin/stderr)
```

Comment:
If we can get the conda-forge recipe up soon, let's move this to the conda list. Otherwise, let's change it in a later PR (I'd like to get the first release candidate out today).